client/public/static/js/recipes/recipe_details.js

```diff
@@ -370,15 +370,21 @@ function RecipeDetailsView() {
     noDietaryText.innerHTML = `No user dietary labels.`;
     dietaryContainer.appendChild(noDietaryText);
 
-    const button = document.createElement("button");
+    const updateRecipeButton = document.createElement("button");
     const recipeName = recipe.recipeName;
-    button.textContent = "Update Recipe";
-    button.classList.add("updateRecipeButton"); // Add a class for styling
-    button.addEventListener("click", function() {
+    updateRecipeButton.textContent = "Update Recipe";
+    updateRecipeButton.classList.add("updateRecipeButton"); // Add a class for styling
+    updateRecipeButton.addEventListener("click", function() {
       document.location.href = "/account/update_recipe?userRecipeName="+recipeName;
     });
     var container = document.getElementById("updateRecipeContainer");
-    container.appendChild(button);
+    container.appendChild(updateRecipeButton);
+
```

Comment:
Could you please add a screenshot of what this looks like on recipe details? 

Also, what happens to the button when we submit? Can we submit multiple times? If so, we should disable the button if it's been "submitted"